Searches/Loopinvariant.java

```diff
@@ -0,0 +1,55 @@
+package algorithm_homework;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.StringTokenizer;
+
+public class Loopinvariant {
+	  public static int search(int[] arr, int target) {
+          int first = 0;
+          int last = arr.length;
+          int mid;
+          while (first <= last) {
+              mid = (first + last) / 2;
+              if (target == arr[mid]) {return mid; }
```

Comment:
Please see the code style conventions for if-else: http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-142311.html#449